index.js

```diff
@@ -424,6 +424,7 @@ var extensions = {
         );
       },
     },
+    'tsx/dist/cjs/index.cjs',
```

Comment:
I added these to the end of the array for given ext. Could you pick up my work? I had some trouble with testing this and the test were very flaky. Thx!